.github/workflows/codeql.yml

```diff
@@ -1,25 +1,12 @@
-# For most projects, this workflow file will not need changing; you simply need
-# to commit it to your repository.
-#
-# You may wish to alter this file to override the set of languages analyzed,
-# or to provide custom queries or build logic.
-#
-# ******** NOTE ********
-# We have attempted to detect the languages in your repository. Please check
-# the `language` matrix defined below to confirm you have the correct set of
-# supported CodeQL languages.
-#
 name: "CodeQL"
 
 on:
   workflow_dispatch:
   push:
-    branches: [ "main" ]
+    branches: [ "main", "*-servicing" ]
   pull_request:
     # The branches below must be a subset of the branches above
-    branches: [ "main" ]
-  schedule:
-    - cron: '27 5 * * 2'
+    branches: [ "main" , "*-servicing"]
```

Comment:
I wouldn't include the servicing branches. In general, CodeQL only needs to be run on the branch with the latest code, the default. If there are critical issues detected by CodeQL, they can be considered for backport. But we don't need to scan old versions of the code.